CMakeLists.txt

```
@@ -181,8 +184,6 @@ add_library( random_generators test/random_generators.cpp test/random_generators
 set_target_properties( opengv random_generators PROPERTIES
                     SOVERSION ${PROJECT_VERSION}
                     VERSION ${PROJECT_VERSION}
-                    CXX_STANDARD 11
-                    CXX_STANDARD_REQUIRED ON
                     DEBUG_POSTFIX d )
```

```suggestion
                     )
```
For `DEBUG_POSTFIX`we can just set a cache variable that initialize `CMAKE_DEBUG_POSTFIX` that will automatically set it for all targets, without doing it explicitly:
```cmake
set(CMAKE_DEBUG_POSTFIX "d" CACHE STRING "The postfix to use for the debug version of the target names")
```